### ISSUE TITLE
Portal Help - search items

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/nmdc-server#357]
Need to address confusion on the difference between terms.

Get descriptions from the NMDC Schema, and flag where descriptions are missing (all terms in the schema should have descriptions, as they are pulled from existing standards/systems). Will require team to put together content.

Priority - medium
Urgency - medium

**Comment by jbeezley** [comment]
I would suggest splitting this into two issues.  One assigned to me to grab the description from the schema and serve the information from the API.  The other to identify and fill in missing descriptions.

**Comment by jbeezley** [comment]
I would guess someone from the AIM 1 team as they are generally responsible for maintaining the schema.

**Comment by jeffbaumes** [comment]
Marking this as small for the portal update process assuming schema descriptions are in place.

**Comment by ssarrafan** [comment]
@dehays do you have a suggestion for who can be assigned the task of identifying and filling in missing descriptions?  The plan is to create a separate GH issue for that part for the next (or some future) sprint.

**Comment by dehays** [comment]
I think there are actually 3 tasks here.  

1) Use the description field of different entity instances to set descriptions visible in UI.  @jbeezley @subdavis 
    Blocked by...

2) All description fields in metadata entity instances have values.  Populating these is neither Jon's or Bill's responsibility.  And this probably requires input from those in analysis.
Blocked by...

3) Add description field to all entities in NMDC Schema that should have descriptions displayed in UI.  @wdduncan 
This follows the discussion here microbiomedata/nmdc-schema#20

**Comment by aclum** [comment]
1. work is still outstanding 
2. Work on this is being done in https://github.com/microbiomedata/nmdc-schema/issues/1306
3. Montana has been working on making sure all the schema fields are described in https://github.com/microbiomedata/nmdc-schema/issues/685

**Comment by aclum** [comment]
Closing this in favor of the individual ticketes.
